crates/egui/src/ui.rs

```diff
@@ -544,6 +544,11 @@ impl Ui {
         self.painter().layer_id()
     }
 
+    /// Returns the `FontId` corresponding to the given `TextStyle`.
+    pub fn font_id_by_text_style(&self, style: &TextStyle) -> FontId {
```

Comment:
I would suggest to call it `font_id()` or `text_style_font_id` this would match with `text_style_height`

Reply:
thank you.
Both `font_id()` and `text_style_font_id()` looks good.
I'll have to wait for emilk's opinion.